Update repositories.txt

csobirka opened the pull request.

per1234 reviewed (commented):
Closing as duplicate of https://github.com/arduino/library-registry/pull/6013

@csobirka please don't submit duplicate pull requests.

---

Perhaps you actually intended to accomplish something productive with this pull request? If so, I am happy to provide advice on how you can accomplish whatever goal you might have. But I can only do that if you make the effort to clearly communicate your intent. You can do that by adding a detailed description in a reply here on the pull request thread. I'll continue to monitor the thread even though the PR is closed.

Although we have automated the library submission process, all other maintenance of this repository is done by humans. So when you want to accomplish something here other than submitting a library you must communicate in a way that a human can understand.

If English is not your native language, you are welcome to compose the description in your native language and then use an automatic translation service (e.g., [**Google Translate**](https://translate.google.com/)) to convert it to English language before posting it in a comment.

csobirka commented:
Dear @per1234,

I have modified the BasicExample.ino, and the .cpp and .h files, because I added new functions for the library (csobirka/SevenSegment repository, v1.1.4). 

The main reason for my action is that I didn't manage to figure out how to update the accessibility in Arduino Library Manager. I would like to ask how to update it in the future or how does the system update it? Will it be automatically refreshed?

Sorry for the duplication, it was not on purpose.

Thanks your assistance in advance!

Yours sincerely,
csobirka

per1234 commented:
Follow the instructions here to publish a new release of the library with the updates:

https://github.com/arduino/library-registry/blob/main/FAQ.md#how-can-i-publish-a-new-release-once-my-library-is-in-the-list

Note that you only need to make a new [release](https://docs.github.com/en/github/administering-a-repository/releasing-projects-on-github/managing-releases-in-a-repository) or [tag](https://git-scm.com/docs/git-tag) in your library repository to publish a new release to Library Manager. It is not necessary to submit a pull request to this repository in order to publish a release.

It does take some time for the new release to be indexed, and for the updated index to propagate through the cache. So please be patient after making your release. We give an estimate of availability within one day's time.

If you want to check on the indexing of your library, you can check the dedicated logs page:

https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

If you have any questions or concerns, please let me know.

Regards, Per
